src/app/gps/qgsgpsinformationwidget.h

```diff
@@ -105,6 +116,14 @@ class QgsGPSInformationWidget: public QWidget, private Ui::QgsGPSInformationWidg
     QFile *mLogFile = nullptr;
     QTextStream mLogFileTextStream;
     QColor mTrackColor;
+    QIntValidator *acquisitionIntValidator;
+    QIntValidator *distanceThresholdValidator;
+    QLineEdit *acIntervalEdit, *distThresholdEdit;
+    nmeaPOS lastNmeaPosition;
+    QTimer *acquisitionTimer;
```

Comment:
std::unique_ptr this to avoid explicit delete in destructor

Reply:
thanks for your useful suggestions.